weblate/trans/management/commands/list_file_format_params.py

```diff
@@ -0,0 +1,74 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import operator
+from collections.abc import Iterable
+from itertools import zip_longest
+
+from weblate.trans.file_format_params import FILE_FORMATS_PARAMS
+from weblate.utils.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "List file format parameters"
+
+    def format_file_formats(self, file_formats: Iterable[str]) -> list[str]:
+        return [
+            " ".join([f"``{f}``" for f in file_formats[i * 4 : (i + 1) * 4]])
+            for i in range((len(file_formats) // 4) + 1)
+        ]
+
+    def get_help_text(self, param) -> list[str]:
+        result = []
+        if param.help_text:
+            result.append(str(param.help_text))
+        if param.choices:
+            if result:
+                result.append("")
+            result.append("Available choices:")
+            for value, description in param.choices:
+                result.extend(
+                    (
+                        "",
+                        f"``{value}``".replace("\\", "\\\\"),
+                        f"  {description}".replace("\\", "\\\\"),
+                    )
+                )
+        return result
+
+    def handle(self, *args, **options) -> None:
+        """List file format parameters."""
```

Comment:
Please add note about generated content here (and include in the rst file), so that it is clear that the rst file should not be edited manually. Something like:

https://github.com/WeblateOrg/weblate/blob/bae7bfd437602955b690a53da9285f4a561e1d09/weblate/addons/management/commands/list_addons.py#L69-L71